packages/controls/src/controls/font/font.ts

```diff
@@ -94,11 +89,15 @@ class Definition<C extends Config> extends ControlDefinition<
       value: withoutVariants,
     })
 
-    const schemas = <V, D>(value: SchemaType<V>, data: SchemaType<D>) => {
+    const schemas = <V, D, V2 extends ZodLiteral<boolean>>(
+      value: SchemaType<V>,
+      data: SchemaType<D>,
+      variant: V2,
```

Comment:
This is the critical difference. I'm using literal boolean values instead and changing the config type based on your input.